docs/src/main/paradox/index.md

```diff
@@ -0,0 +1,153 @@
+| Akka Persistence plugin for R2DBC                | $akka-persistence-r2dbc.version$     | [Documentation](https://doc.akka.io/docs/akka-persistence-r2dbc/current/)                                                                                                                                                                                                                                              |
+| Akka Projections                      | $akka-projections.version$           | [Documentation](https://doc.akka.io/docs/akka-projection/current/)                                                                                                                                                                                                                                                     |
+| Akka Management                       | $akka-management.version$            | [Documentation](https://doc.akka.io/docs/akka-management/current/)                                                                                                                                                                                                                                                     |
+| Akka Diagnostics                      | $akka-diagnostics.version$           | [Documentation](https://doc.akka.io/docs/akka-diagnostics/current/)                                                                                                                                                                                                                                                    |
+| Alpakka                               | $alpakka.version$                    | [Documentation](https://doc.akka.io/docs/alpakka/current/)                                                                                                                                                                                                                                                             |
+| Alpakka Kafka                         | $alpakka-kafka.version$              | [Documentation](https://doc.akka.io/docs/alpakka-kafka/current/)                                                                                                                                                                                                                                                       |
+
+## Akka (core) $akka.version$  
+At the core of Akka: A model for concurrency and distribution without all the pain of threading primitives.
+
+[Documentation](https://doc.akka.io/docs/akka/current/) [JavaDoc](https://doc.akka.io/japi/akka/current/index.html) [ScalaDoc](https://doc.akka.io/api/akka/current/akka/)
+
```

Comment:
In an earlier draft you had a page dependencies-listing.html that showed all artifact names in copy paste friendly build formats for maven, gradle, sbt. I think that would be useful.

Maybe it could be incorporated in this page?

```
## Akka (core)
...
### Akka Actors
...
### Akka Cluster
...
...
### Dependencies listing
maven/gradle/sbt
...
## Akka gRPC
...
### Dependencies listing
maven/gradle/sbt
```

Reply:
Depends on what the purpose of the page is. We have better pages for brief overview, such as the Akka docs landing page.

I see the purpose as:
* what is the latest version? and which versions should I use together? (the table answers that)
* what shall I paste in the build to add x? (I already know how to use the feature and I don't want to browse various reference doc pages to find it)